### PR TITLE
[en] typo fix "privilge -> privilege"

### PR DESCRIPTION
**Pull request by kardashov**

### CODE REVIEW

**Comment by AmanSarraf** [comment]
Thanks for your contribution, @kardashov .
can we review and merge this @bradtopol 
/assign

File: content/en/docs/concepts/security/rbac-good-practices.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ weight: 60
 Kubernetes {{< glossary_tooltip text="RBAC" term_id="rbac" >}} is a key security control 
 to ensure that cluster users and workloads have only the access to resources required to 
 execute their roles. It is important to ensure that, when designing permissions for cluster
-users, the cluster administrator understands the areas where privilge escalation could occur, 
+users, the cluster administrator understands the areas where privilege escalation could occur,
```
Comment (AmanSarraf) [inline content/en/docs/concepts/security/rbac-good-practices.md:15]: good work

**Review by bishal7679 — APPROVED** [review]
@kardashov Nice catch!
/lgtm

**Pull request merged**